Source/Scene/Scene.js

```diff
@@ -1867,6 +1867,9 @@ define([
             }
 
             clearDepth.execute(context, passState);
+            if (context.stencilBuffer) {
+                scene._stencilClearCommand.execute(context, passState);
+            }
```

Comment:
Based on offline discussion, this clear is no longer needed since the clear command issued by the tileset will be run in each frustum.